src/clj_zipkin/tracer.clj

```diff
@@ -68,21 +53,21 @@
   ([span host trace-id span-id parent-id start finish & [annotations]]
      (let [endpoint (host->endpoint host)
            start-timestamp (* 1000 (time-coerce/to-long start))
-           start-annotation (Annotation. start-timestamp
-                                         (str "start:" (name span)) endpoint 0)
-           finish-annotation (Annotation. (* 1000 (time-coerce/to-long finish))
-                                          (str "end:" (name span)) endpoint 0)
-           annotations (for [[k v] annotations]
-                         (Annotation. start-timestamp
-                                      (str (name k) ":" (name v)) endpoint 0))]
+           end-timestamp (* 1000 (time-coerce/to-long finish))
+           binary-annotations (if (empty? annotations)
+                                [(BinaryAnnotation/create "lc" *default-component-name* endpoint)]
+                                (for [[k v] annotations]
+                                  (BinaryAnnotation/create (name k) (name v) endpoint)))]
```

Comment:
key/value are BinaryAnnotation in zipkin. Annotations should be loggable, but they'd need a timestamp.